src/System.Collections/tests/project.json

```diff
@@ -27,7 +27,9 @@
       "dependencies": {
         "System.Runtime.Serialization.Formatters": "4.3.0-beta-24522-03"
       }
-    }
+    },
+    "netcoreapp1.1": {},
+    "netcoreapp1.0": {}
```

Comment:
You can remove the netcoreapp1.0 framework as you aren't building this project for that target framework.

Once you add the build configuration I suggested you should be able to test the build and run the tests for netcoreapp1.1. I suspect you will need to add:
```
 "System.Runtime.Serialization.Formatters": "4.3.0-beta-24522-03"
```
Under your netcoreapp1.1 section here as well, in order to get that dependency passed for the netcoreapp1.1 build.